web/packages/teleterm/src/ui/ConnectMyComputer/connectMyComputerContext.tsx

```diff
@@ -239,22 +262,35 @@ export const ConnectMyComputerContextProvider: FC<{
         );
       } catch (e) {
         if (isAccessDeniedError(e)) {
-          ctx.notificationsService.notifyInfo({
-            title: 'The node may be visible for a few more minutes.',
-            description:
-              'You do not have permissions to remove nodes, but it will be removed automatically.',
-          });
+          hasAccessDeniedError = true;
         } else {
           throw e;
         }
+        ctx.notificationsService.notifyInfo(
```

Comment:
This should be moved outside of the `catch` block.

Could you maybe write a test for this behavior specifically, that is sending one notification in one case and another under some different circumstances? It's one of those little things that's easy to miss without tests.

Reply:
Right, it should be outside of the `catch`. Tests added.